### ISSUE TITLE
ERROR | Double space found

### ISSUE THREAD

**Issue by icex** [cakephp/cakephp-codesniffer#275]
Hello,

I think there's a possible bug here:
https://github.com/cakephp/cakephp-codesniffer/blob/a2abf9fd074d49616942b327640bd23ea34bc827/CakePHP/Sniffs/WhiteSpace/TabAndSpaceSniff.php#L62

If my code has 4 whitespaces this will trigger an error..

The Cookbook recommended indentation: https://book.cakephp.org/3/en/contributing/cakephp-coding-conventions.html#indentation

Also PSR-2:
https://book.cakephp.org/3/en/contributing/cakephp-coding-conventions.html#indentation

**Comment by markstory** [comment]
Could you give us an example of the code that is triggering the error you think is incorrect?

**Comment by icex** [comment]
Looks like it doesn't like the way PHPStorm aligns array signs..
```php
<?php

$options = [
    'aaa'          => 'bbb',
    'dddasdfasdfa' => 'asdfsdf',
];
```
```
FILE: src/test.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 4 | ERROR | Double space found
----------------------------------------------------------------------


```

**Comment by dereuromark** [comment]
Such alignment is not valid, you must never align code this way.

**Comment by markstory** [comment]
Closing as the ruleset is working correctly. There is more than one space due to the alignment.

This ruleset flags that as an error because we don't encourage aligning operators as it often increases diff sizes by changing lines that otherwise would be unchanged.
